src/k8s-engine/k8s.ts

```diff
@@ -40,6 +40,9 @@ export interface KubernetesBackend extends events.EventEmitter {
   /** The amount of memory in the VM, in MiB, or 0 if the VM is not running. */
   memory: Promise<number>;
 
+  /** The port the Kubernetes server listens on (default 6443) */
```

Comment:
Wouldn't that be `… server _is currently_ listening on`, with no default?  Also, I guess we should make this readonly?

Reply:
Done